rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -201,35 +228,46 @@ private function diffFullyQualifiedObjectTypes(array $mainTypes, array $typesToR
 
     /**
      * @param array<AliasedObjectType|FullyQualifiedObjectType> $useImportTypes
-     * @param array<FullyQualifiedObjectType|AliasedObjectType> $constantUseImportTypes
      * @param array<FullyQualifiedObjectType|AliasedObjectType> $functionUseImportTypes
+     * @param array<FullyQualifiedObjectType|AliasedObjectType> $constantUseImportTypes
      * @return Use_[]
      */
     private function createUses(
         array $useImportTypes,
-        array $constantUseImportTypes,
         array $functionUseImportTypes,
+        array $constantUseImportTypes,
         ?string $namespaceName
     ): array {
         $newUses = [];
 
         /** @var array<Use_::TYPE_*, array<AliasedObjectType|FullyQualifiedObjectType>> $importsMapping */
         $importsMapping = [
             Use_::TYPE_NORMAL => $useImportTypes,
-            Use_::TYPE_CONSTANT => $constantUseImportTypes,
             Use_::TYPE_FUNCTION => $functionUseImportTypes,
+            Use_::TYPE_CONSTANT => $constantUseImportTypes,
         ];
 
         foreach ($importsMapping as $type => $importTypes) {
-            /** @var AliasedObjectType|FullyQualifiedObjectType $importType */
```

Comment:
This type hint is not needed, the type of this variable can be guesses from the type hints for the params of the function